src/interfacers/EmonHubEmoncmsHTTPInterfacer.py

```diff
@@ -138,5 +151,13 @@ def set(self, **kwargs):
                 self._log.info("Setting " + self.name + " sendstatus: " + setting)
                 self._settings[key] = setting
                 continue
+            elif key == 'compress':
+                self._log.info("Setting " + self.name + " compress: " + setting)
+                if setting == "1":
+                    setting = True
+                else:
+                    setting = False
+                self._settings[key] = setting
```

Comment:
These 5 lines could just be
`self._settings[key] = bool(setting)`
if `setting` is indeed an int, but I think it will be a string in general so probably
`self._settings[key] = bool(int(setting))`